src/test/accounts.test.ts

```diff
@@ -0,0 +1,173 @@
+import { EasyPrivateVotingContractArtifact, EasyPrivateVotingContract } from "../artifacts/EasyPrivateVoting.js"
+import { AccountManager, AccountWallet, CompleteAddress, ContractDeployer, createLogger, Fr, PXE, waitForPXE, TxStatus, createPXEClient, getContractInstanceFromDeployParams, Logger } from "@aztec/aztec.js";
+import { getInitialTestAccountsWallets, generateSchnorrAccounts } from "@aztec/accounts/testing"
+import { getSchnorrAccount } from '@aztec/accounts/schnorr';
+import { spawn } from 'child_process';
+import { SponsoredFeePaymentMethod } from '../utils/sponsored_fee_payment_method.js';
+import { getFeeJuiceBalance, type L2AmountClaim, L1FeeJuicePortalManager, FeeJuicePaymentMethodWithClaim, AztecAddress } from "@aztec/aztec.js";
+import { createEthereumChain, createL1Clients } from '@aztec/ethereum';
+
+const setupSandbox = async () => {
+    const { PXE_URL = 'http://localhost:8080' } = process.env;
+    const pxe = createPXEClient(PXE_URL);
+    await waitForPXE(pxe);
+    return pxe;
+};
+const sleep = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
+
+describe("Accounts", () => {
+    let pxe: PXE;
+    let wallets: AccountWallet[] = [];
+    let accounts: CompleteAddress[] = [];
+    let logger: Logger;
+    let sandboxInstance;
+    let sponsoredPaymentMethod: SponsoredFeePaymentMethod;
+
+    let randomAccountManagers: AccountManager[] = [];
+    let randomWallets: AccountWallet[] = [];
+    let randomAddresses: AztecAddress[] = [];
+
+    let l1PortalManager: L1FeeJuicePortalManager;
+    let feeJuiceAddress: AztecAddress;
+    let skipSandbox: boolean;
+
+    beforeAll(async () => {
+        skipSandbox = false;
```

Comment:
Is this meant to updated in the script itself? wdyt about being able to pass it as a flag? just thinking that it would be more clear in the command to run the test if there was something like `SPAWN_SANDBOX=true yarn test`